content/blog/announcing-kudo-0.6.0.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2019-09-09
+---
+
+# Announcing KUDO 0.6.0
+
+We are proud to announce the release of KUDO v0.6.0! 
+
+## Release Highlights
```

Comment:
```suggestion
<!-- more -->

## Release Highlights
```

Reply:
We **need** the `<!-- more -->` comment to have VuePress render the above section an an excerpt; otherwise the build fails (see the failed netlify deploy attempts).